README.md

```diff
@@ -248,6 +248,18 @@ if:
     - "label-1"
     - "label-2"
       
+  # "repository" is satisfied if the pull request repository matches any one of the
+  # patterns within the "matches" list, and does not match all of the patterns
+  # within the "not_matches" list.
```

Comment:
I don't think this behavior is what is actually implemented. Based on the code and tests, I believe the `matches` list always wins: if the repository matches, the predicate is satisfied. Looking at the existing `title` predicate, I see its behavior also doesn't match what the docs say.

Unless you want to change the actual behavior, let's update this comment to be:

```
# "repository" is satisfied if the pull request repository matches any one of the
# patterns within the "matches" lis or does not match all of the patterns
# within the "not_matches" list.
```

If you don't mind also updating the `title` docs, I'd appreciate it.